src/utils/draftutil.js

```diff
@@ -238,25 +244,19 @@ function createPacks(draft, format, seats, nextCardFn) {
 }
 
 // NOTE: format is an array with extra attributes, see getDraftFormat()
-export function populateDraft(format, cards, bots, seats, user) {
+export function createDraft(format, cards, bots, seats, user, seed = false) {
```

Comment:
Please keep PRs more granular, and state all changes in description. There is no indication in the PR description that this PR adds a seed to draft creation. Why do we need this?

Reply:
We have to have a seed to support consistent p1p1 creation for the cube. This was a required change for the stated purpose to work. I can add a description as to how the default format works with p1p1 though.